cnd/src/db/integration_tests.rs

```diff
@@ -1,5 +1,5 @@
 use crate::{
-    asset::{Erc20, Ether},
+    asset::{Bitcoin as BitcoinAsset, Erc20, Ether},
```

Comment:
Prefer `asset::Bitcoin` here instead of the alias.

Reply:
sorry, just saw that now, PR already merged. I remember I had some problems with the macro when using `asset::Bitcoin` at this particular position. If it matters please create a follow up PR :)